docs/2.guide/3.going-further/12.custom-router.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Custom router"
+description: "In Nuxt 3, your routing is the defined by the structure your files inside the pages directory. However, since it uses vue-router under the hood, Nuxt offers you several ways to add custom routes in your project."
```

Comment:
```suggestion
description: "In Nuxt 3, your routing is defined by the structure of your files inside the pages directory. However, since it uses vue-router under the hood, Nuxt offers you several ways to add custom routes in your project."
```